src/screen/GameScreen.java

```diff
@@ -234,13 +234,12 @@ protected final void update() {
 		manageCollisions();
 		cleanBullets();
 		draw();
-
-		// 사용안해서 삭제 게임 클리어 조건
-		//if ((this.enemyShipSet.isEmpty() || this.lives == 0)
-		//		&& !this.levelFinished) {
-		//	this.levelFinished = true;
-		//	this.screenFinishedCooldown.reset();
-		//}
+		// 현재 진행된 시간이 라운드에서 정한 시간과 같으면 클리어로 판단 후 라운드 종료
+		if ((levelTime == this.gameSettings.getRoundTime() || this.lives == 0)
```

Comment:
```suggestion
		if ((levelTime == this.gameSettings.getRoundTime() || this.lives <= 0)
```
현재 라운드 종료 방식이 시간이 끝나거나 남은 생명이 0일 경우로 해두셨는데 지금이야 최대 체력 100에 모든 적들이 20의 데미지를 줘서 5번 충돌 시 끝나지만 만약에 추후 데미지가 다른 적 함선이 생긴다면 남은 체력이 0이 아닌 음수로 갔을 땐 게임이 끝나지 않을 수 있어서 0 이하로 떨어지면 끝내는 조건으로 변경하는게 좋을것 같습니다